src/index.d.ts

```diff
@@ -1249,26 +1229,11 @@ declare namespace admin.machineLearning {
     /** The size of the model. */
     readonly sizeBytes: number;
 
+    /** At most one of these two will be specified. */
```

Comment:
Doesn't look like correct TypeDoc syntax. This will have to be included in the description of the type.

Reply:
I removed the line. Kevin will fix the docs.